tsl/test/sql/dist_compression.sql

```diff
@@ -75,7 +75,13 @@ SELECT * from chunk_compression_stats( 'compressed')
 ORDER BY chunk_name, node_name;
 select * from hypertable_compression_stats('compressed'); 
 
--- Compress twice to generate NOTICE that the chunk is already compressed
+-- Compress twice to notice idempotent operation
+SELECT compress_chunk(chunk, if_not_compressed => true)
+FROM show_chunks('compressed') AS chunk
+ORDER BY chunk
+LIMIT 1;
```

Comment:
This only test that compressing a single chunk return the same result. Since this should return the same result with any  combination of chunks being compressed, I think you need to test with several chunks and first compress some of them (say, every second or third) and then all of them twice below (first time will compress the remaining chunks, second time will compress none) to make sure that you get the same result in both the last calls.